sources/packages/api/src-sql/Student/Add-SINValidation.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS sin_validation_id INT NOT NULL REFERENCES sims.sin_validations(id);
```

Comment:
Is it going to work with existing records?

Reply:
No wil create bulk select and insert and update